drv/lpc55-rng/src/config.rs

```diff
@@ -0,0 +1,11 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#[derive(serde::Deserialize, Default, Debug)]
+#[serde(rename_all = "kebab-case")]
+#[cfg(feature = "dice-seed")]
+pub struct DataRegion {
+    pub address: u32,
```

Comment:
**suggestion:** Use `usize` for these. For `address` you're doing `as *const X` so the type here doesn't really matter, and for `size` you're doing `as usize` anyway.

Of course, the end result is exactly the same either way.